src/main/java/workers/MarkWorker.java

```diff
@@ -0,0 +1,29 @@
+package workers;
+
+import java.util.ArrayList;
+
+import tasks.Task;
+
+/**
+ * A specialised worker class that does the mark command.
+ */
+public class MarkWorker extends TaskWorker {
+    /**
+     * This method marks the Task at the specified index as done.
```

Comment:
Javadoc comments should have an empty line between description and parameter section.